contracts/test/forkMainnet/AMMWrapperWithPath/Setup.t.sol

```diff
@@ -155,21 +156,69 @@ contract TestAMMWrapperWithPath is StrategySharedSetup {
         sig = abi.encodePacked(r, s, v, bytes32(0), uint8(2));
     }
 
+    function _createSpenderPermitFromOrder(AMMLibEIP712.Order memory defaultOrder)
+        internal
+        view
+        returns (SpenderLibEIP712.SpendWithPermit memory takerAssetPermit)
+    {
+        // Declare the 'userAddr sends takerAssetAmount amount of takerAssetAddr to AMM contract'
+        // SpendWithPermit struct from defaultOrder parameter
+        takerAssetPermit = SpenderLibEIP712.SpendWithPermit(
+            defaultOrder.takerAssetAddr,
+            address(ammWrapperWithPath),
+            defaultOrder.userAddr,
+            address(ammWrapperWithPath),
+            defaultOrder.takerAssetAmount,
+            AMMLibEIP712._getOrderHash(defaultOrder),
+            uint64(defaultOrder.deadline)
+        );
+        return takerAssetPermit;
+    }
+
+    function _signSpendWithPermit(
+        uint256 privateKey,
+        SpenderLibEIP712.SpendWithPermit memory spendWithPermit,
+        SignatureValidator.SignatureType sigType
+    ) internal returns (bytes memory sig) {
+        uint256 SPEND_WITH_PERMIT_TYPEHASH = 0x52718c957261b99fd72e63478d85d1267cdc812e8249f5a2623566c1818e1ed0;
+        bytes32 structHash = keccak256(
+            abi.encode(
+                SPEND_WITH_PERMIT_TYPEHASH,
+                spendWithPermit.tokenAddr,
+                spendWithPermit.requester,
+                spendWithPermit.user,
+                spendWithPermit.recipient,
+                spendWithPermit.amount,
+                spendWithPermit.actionHash,
+                spendWithPermit.expiry
+            )
+        );
+        bytes32 spendWithPermitHash = getEIP712Hash(spender.EIP712_DOMAIN_SEPARATOR(), structHash);
+        if (sigType == SignatureValidator.SignatureType.Wallet) {
+            (uint8 v, bytes32 r, bytes32 s) = vm.sign(privateKey, ECDSA.toEthSignedMessageHash(spendWithPermitHash));
+            sig = abi.encodePacked(r, s, v, uint8(sigType)); // new signature format
+        } else {
+            (uint8 v, bytes32 r, bytes32 s) = vm.sign(privateKey, spendWithPermitHash);
+            sig = abi.encodePacked(r, s, v, uint8(sigType)); // new signature format
+        }
+    }
+
```

Comment:
This function is no longer used, right?

Reply:
Yes! I ignored ammWrapper while following ammWrapperWithPath still has the same function, and it has been removed.